tools/docker/docker-images.md

```diff
@@ -6,44 +6,91 @@ order: 4
 
-It features `fedora:latest` tag for rawhide and `fedora:22` tag for Fedora 22.
+Docker images in `fedora/` namespace feature `fedora:latest` tag for rawhide and `fedora:22` tag for Fedora 22.
 
-To get Fedora 22 image, run:
+To get Fedora 22 base image, run:
 
 ```
+Pull the image from docker.io
 $ sudo docker pull fedora:22
+Run some command from the image
+$ sudo docker run --rm -ti fedora:22 bash
+```
+
+There are also a lot of application Docker images built as layered images on top of Fedora base image. It's sources live in [Fedora Dockerfiles repository](https://github.com/fedora-cloud/Fedora-Dockerfiles) and are available under `fedora/` namespace on the Docker Hub.
+
+For example, to pull and run the MariaDB Docker container, run:
+```
+Pull the image from docker.io
+$ sudo docker pull fedora/mariadb
+Run the container
+$ sudo docker run fedora/mariadb
+fedora/mariadb
 ```
 
+The list of available Fedora Docker images is 
+
 ## CentOS images
 
-Here is the [official CentOS repository](https://hub.docker.com/_/centos/).
+You can find find all the official Docker images provided by CentOS community in the [official CentOS repository](https://hub.docker.com/u/centos/) and the base Docker image in the [official library](https://hub.docker.com/_/centos/) on Docker Hub.
 
-To get CentOS 7 image, run:
+To get CentOS 7 base image, run:
 
 ```
+Pull the image from docker.io
 $ sudo docker pull centos:7
+Run some command from the image
+$ sudo docker run --rm -ti centos:7 bash
 ```
 
-To get CentOS 6 image, run:
+To get CentOS 6 base image, run:
 
 ```
+Pull the image from docker.io
 $ sudo docker pull centos:6
+Run some command from the image
+$ sudo docker run --rm -ti centos:6 bash
 ```
 
 There is always `centos:latest` tag for the latest released version.
 
 ### Software Collections based images
 
-You can also get Docker images based on [Software Collections](https://www.softwarecollections.org/en/) which are currently released under [the OpenShift organization](https://hub.docker.com/u/openshift/). Some of them are enabled for [Source-To-Image](https://github.com/openshift/source-to-image).
-
-To download them just run `docker pull openshift/IMAGE_NAME`.
+The [official CentOS repository](https://hub.docker.com/u/centos/) contains Docker images that are similar to the images provided by Red Hat under `rhscl/` namespace.
+
+These Docker images are based on [Software Collections](https://www.softwarecollections.org/en/). Some of them (older versions) are released under [the OpenShift organization](https://hub.docker.com/u/openshift/), the newer versions are available under [CentOS organization](https://hub.docker.com/u/centos/). Some of them are enabled for [Source-To-Image](https://github.com/openshift/source-to-image).
+
+To download them just run `docker pull openshift/IMAGE_NAME` or `docker pull centos/IMAGE_NAME`.
```

Comment:
Since you are listing full names in the table, I would change it to `docker pull IMAGE_NAME`.